src/components/Select.tsx

```diff
@@ -53,7 +53,6 @@ const Select: React.SFC<SelectProps> = props => {
                 required={required}
                 message={isOpen ? " " : ""}
                 errorMessage={errorMessage}
-                readOnly
```

Comment:
@ritz078 It made the font color gray, which was not according to the design system.